dio/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 - Make `LogInterceptor` prints in DEBUG mode (when the assertion is enabled) by default.
 - Fix `IOHttpClientAdapter.onHttpClientCreate` Repeated calls
+- `IOHttpClientAdapter.onHttpClientCreate` has been deprecated because of unclear handling
```

Comment:
Can we keep the CHANGLOG with critical info only and move details into the migration guide?

Reply:
We only have breaking information there